h5p-bildetema/src/components/Language/Language.module.scss

```diff
@@ -0,0 +1,76 @@
+.language {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 1.5rem;
+}
+
+.input_container {
+  border: 1px solid red;
+  position: relative;
+  padding-left: 1rem;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+.input_container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.checkmark {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  box-shadow: 0 0 0 3px black;
+  border-radius: 5px;
+}
+
+.input_container input:checked ~ .checkmark {
+  background-color: black;
+}
+
+.checkmark:after {
+  content: "";
+  position: absolute;
+  display: none;
+}
+
+.input_container input:checked ~ .checkmark:after {
+  display: block;
+}
+
+.input_container .checkmark:after {
+  left: 4px;
+  top: 1px;
+  width: 5px;
+  height: 10px;
+  border: solid white;
+  border-width: 0 3px 3px 0;
+  -webkit-transform: rotate(45deg);
+  -ms-transform: rotate(45deg);
+  transform: rotate(45deg);
+}
+
+.language_label {
+  font-size: 1.2rem;
+  cursor: pointer;
+  background-color: transparent;
+  background-repeat: no-repeat;
+  border: none;
+  overflow: hidden;
+  outline: none;
```

Comment:
🚩🚩🚩🚩🚩🚩🚩🚩🚩🚩

This component is not very keyboard friendly. It does work, but it's impossible for the user to see where they've tabbed to. Consider adding the orange outline we've talked about to the entire label.